c++/test/TestWriter.cc

```diff
@@ -16,19 +16,23 @@
  * limitations under the License.
  */
 
+#include <_types/_uint64_t.h>
+#include "gtest/gtest.h"
```

Comment:
These are not required, I think

Reply:
They were auto added by clangd, removed.